0-common/schema-ddl/.release.yml

```diff
@@ -0,0 +1,42 @@
+# Required: local settings
```

Comment:
added this file because in the issue there was a phrase about removing bintray sbt plugin. This file is used in snowplow-elasticsearch-loader . So I modified it for this project.